lnrpc/walletunlocker.proto

```diff
@@ -188,6 +197,63 @@ message InitWalletResponse {
     bytes admin_macaroon = 1;
 }
 
+message WatchOnly {
+    /*
+    The unix timestamp in seconds of when the master key was created. lnd will
+    only start scanning for funds in blocks that are after the birthday which
+    can speed up the process significantly. If the birthday is not known, this
+    should be left at its default value of 0 in which case lnd will start
+    scanning from the first SegWit block (481824 on mainnet).
+    */
+    uint64 master_key_birthday_timestamp = 1;
+
+    /*
+    The fingerprint of the root key (also known as the key with derivation path
+    m/) from which the account public keys were derived from. This may be
+    required by some hardware wallets for proper identification and signing. The
+    bytes must be in big-endian order.
+    */
+    bytes master_key_fingerprint = 2;
```

Comment:
Is there a strong requirement that this is on this level? In that, can it go one layer below within the `WatchOnlyAccount` struct?